### PR TITLE
Add new package: marv-pg-driver

### PR DESCRIPTION
**Pull request by rjoonas**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/marv-pg-driver/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Type definitions for marv-pg-driver 4.0
+// Project: https://github.com/guidesmiths/marv-pg-driver
+// Definitions by: Joonas Rouhiainen <https://github.com/me>
```
Comment (DangerBotOSS) [inline types/marv-pg-driver/index.d.ts:3]: - :no_entry_sign: This line should have your github username in it, not /me

**Review by domdomegg — APPROVED** [review]

**Review by Grayson-code — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
It has no dependency at runtime to 'marv':
https://github.com/guidesmiths/marv-pg-driver/blob/master/package.json#L28-L31
One should not change this, this will effectively force unrelated package, am I seeing this correctly?

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by rjoonas** [comment]
> It has no dependency at runtime to 'marv': https://github.com/guidesmiths/marv-pg-driver/blob/master/package.json#L28-L31 One should not change this, this will effectively force unrelated package, am I seeing this correctly? You want to reuse interface from 'marv', is that correct?

Fair point, @peterblazejewicz. On the other hand, the package exists only to implement a standardized interface as defined by the parent library. I don't see realistic use cases where anyone would install and use it as a standalone module without the parent library.

I believe it's better that these plugin-like add-on modules are typechecked against a single interface defined in the parent module, instead of copy-pasting the same type to all plugin modules.

**Comment by rjoonas** [comment]
Ready to merge

**Pull request merged**